crates/hyperswitch_connectors/src/connectors/nomupay/transformers.rs

```diff
@@ -93,136 +295,310 @@ impl TryFrom<&ConnectorAuthType> for NomupayAuthType {
+                    message: format!("{:?} is not supported", other_bank),
+                    connector: "nomupay",
+                }
+                .into()),
+            },
+            _ => Err(errors::ConnectorError::NotImplemented(
+                "This payment method is not implemented for Nomupay".to_string(),
+            )
+            .into()),
         }
     }
 }
 
-//TODO: Fill the struct with respective fields
-#[derive(Default, Debug, Clone, Serialize, Deserialize)]
-pub struct RefundResponse {
-    id: String,
-    status: RefundStatus,
-}
-
-impl TryFrom<RefundsResponseRouterData<Execute, RefundResponse>> for RefundsRouterData<Execute> {
+// PoRecipientAccount response
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<PayoutsResponseRouterData<F, OnboardTransferMethodResponse>>
+    for PayoutsRouterData<F>
+{
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(
-        item: RefundsResponseRouterData<Execute, RefundResponse>,
+        item: PayoutsResponseRouterData<F, OnboardTransferMethodResponse>,
     ) -> Result<Self, Self::Error> {
         Ok(Self {
-            response: Ok(RefundsResponseData {
-                connector_refund_id: item.response.id.to_string(),
-                refund_status: enums::RefundStatus::from(item.response.status),
+            response: Ok(PayoutsResponseData {
+                status: Some(PayoutStatus::RequiresCreation),
+                connector_payout_id: Some(item.response.id),
+                payout_eligible: None,
+                should_add_next_step_to_process_tracker: false,
+                error_code: None,
+                error_message: None,
             }),
             ..item.data
         })
     }
 }
 
-impl TryFrom<RefundsResponseRouterData<RSync, RefundResponse>> for RefundsRouterData<RSync> {
+// PoFulfill Request
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<(&PayoutsRouterData<F>, FloatMajorUnit)> for NomupayPaymentRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(
-        item: RefundsResponseRouterData<RSync, RefundResponse>,
+        (item, amount): (&PayoutsRouterData<F>, FloatMajorUnit),
     ) -> Result<Self, Self::Error> {
+        let nomupay_auth_type = NomupayAuthType::try_from(&item.connector_auth_type)?;
+
         Ok(Self {
-            response: Ok(RefundsResponseData {
-                connector_refund_id: item.response.id.to_string(),
-                refund_status: enums::RefundStatus::from(item.response.status),
-            }),
-            ..item.data
+            source_id: nomupay_auth_type.eid,
+            destination_id: item.request.clone().connector_payout_id,
+            payment_reference: item.request.clone().connector_payout_id,
+            amount,
+            currency_code: item.request.destination_currency,
+            purpose: "OTHER".to_string(),
+            description: "This a test payment".to_string(),
```

Comment:
Is this a required field? We can use this if it's optional
```suggestion
            description: item.description,
```

Reply:
done